src/lib/github/getFilesOnInstall.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Gets files that have changed in the last  commits
+ */
+import { Context } from "./types";
+import { getBasicRepoProps } from "./utils";
+
+export interface ModifiedFile {
+  name: string;
+  downloadUrl: string;
+  htmlUrl: string;
+}
+
+export default async function getFiles(context: Context): Promise<ModifiedFile[]> {
+  const { owner, repo } = getBasicRepoProps (context);
+  const octokit = context.github;
+  const committedArray = await octokit.repos.getCommit({owner, repo});
```

Comment:
wasn't this supposed to be `octokit.repos.getCommits` https://octokit.github.io/rest.js/#api-PullRequests-getCommits